src/pages/registration/contributors_tab/Contributors.tsx

```diff
@@ -112,62 +116,59 @@ export const Contributors = ({ contributorRole = ContributorRole.CREATOR, push,
           type: BackendTypeNames.ORGANIZATION,
           id: unitUri,
         })),
-        role: contributorRole,
+        role,
         sequence: relevantContributors.length + 1,
       };
       push(newContributor);
     } else {
       const verifiedContributor: Contributor = {
         ...relevantContributors[unverifiedContributor.index],
-        role: contributorRole,
+        role,
         identity,
       };
       replace(unverifiedContributor.index, verifiedContributor);
     }
   };
 
+  const contributorRole = contributorRoles.length === 1 ? contributorRoles[0] : 'OtherContributor';
```

Comment:
Ser vi har en enum `ContributorRole` som har en `Other`-verdi. Men det er ikke det samme som denne?

Reply:
nei, denne er litt snål men fant ikke noen vei utenom.
`ContributorRole` har alle rolle-typene som kan sette for bidragsytere.
Denne consten brukes for å finne riktig oversettelse. Hvis det er en rolle bruker vi den, men hvis det kan være flere så skal det være "Bidragsytere" og "Legg til bidragsytere" i stedet for "Forfatter" og "Legg til forfatter" feks. 

Så "OtherContributor" her brukes bare for at man skal komme i `default`-en i switchene som henter oversettelser. kunne like godt vært "osteloff"

Reply:
ahh, skjønner 👍